test/validators.js

```diff
@@ -4698,12 +4704,54 @@ describe('Validators', () => {
       valid: ['abc', 'de', 'a', ''],
       invalid: ['abcd'],
     });
+    test({
+      validator: 'isLength',
+      args: [{ max: 6, exact: 5 }],
+      valid: ['abcde', 'fffde', 'allkk'],
+      invalid: ['abcd', 'abcdef', 'vfd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ max: 6, exact: { } }],
+      valid: [],
+      invalid: ['abcd', 'abcdef', 'vfd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ min: 2, max: 6, exact: 5 }],
+      valid: ['abcde', 'fffde', 'allkk'],
+      invalid: ['bsa', 'vfvd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ exact: 2 }],
+      valid: ['fg', 'ff', 'po'],
+      invalid: ['bsa', 'vfvd', '', 'k'],
+    });
     test({
       validator: 'isLength',
       args: [{ max: 0 }],
       valid: [''],
       invalid: ['a', 'ab'],
     });
+    test({
+      validator: 'isLength',
+      args: [{ min: 5, max: 10, exact: [6, 8, 9] }],
+      valid: ['helloguy', 'shopping', 'validator', 'length'],
+      invalid: ['abcde', 'abcdefg'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ min: 5, max: 10, exact: { first: 6, second: 8, third: 9 } }],
+      valid: ['helloguy', 'shopping', 'validator', 'length'],
+      invalid: ['abcde', 'abcdefg', 'abcdefghij'],
+    });
+    test({
```

Comment:
We can remove this test since it tests the same as the test of `{ exact: 2 }`.

Reply:
Or adjust this test to test the combination of a `min` and `exact` values.

Reply:
Actually, the later one with `exact: { first: 6, second: 8, third: 9 }` checks for the condition involving the `exact` being an object. Hence, I have adjusted the previous one with `{exact: 2}` to test the combination of a `min` and `exact` values.

Reply:
I was wrong about this test, the difference is that this is using a string of '9' instead of the number 9. That makes this a different test to `{ exact: 2 }`. So the combination of `min` and `exact` should be an additional test

Reply:
So I have included the combination of a `min` and `exact` values in a test with `args: [{ min: 1, exact: 2 }]`.